cps/pom.xml

```diff
@@ -77,9 +77,27 @@
             <artifactId>groovy-cps</artifactId>
             <version>1.5</version>
         </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.ui</groupId>
+            <artifactId>jquery-detached</artifactId>
+            <version>1.1</version>
+        </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.ui</groupId>
+            <artifactId>ace-editor</artifactId>
+            <version>0.9</version>
+        </dependency>
     </dependencies>
     <!-- TODO currently job depends on tests from here; this dependency should probably be broken -->
```

Comment:
Okay sorry, I see what you mean. It seems like the comment was in the wrong place to begin with. If it's relating to the plugin, then it should be beside it and not floating out here in no mans land.